src/endpoint/s3/ops/s3_get_bucket.js

```diff
@@ -84,6 +84,12 @@ async function get_bucket(req) {
     };
 }
 
+function get_object_owner(params) {
+    return Object.freeze({
```

Comment:
As per today's code and according to [AWS](https://docs.aws.amazon.com/AmazonS3/latest/API/API_Owner.html), owner has also ID - 
const DEFAULT_S3_USER = Object.freeze({
    ID: '123',
    DisplayName: 'NooBaa'
});

ID is missing in the new response

Reply:
why are we not getting this info from req.object_sdk.requesting_account?? we should have both id and name there. and in case of anonymous request we should check how aws returns it.

Reply:
ah sorry i mixed up the requesting user and the object owner... so instead, how about we let the namespace return this info in obj.owner, and then we can really consider how to store and retrieve this info correctly per ns type.

Reply:
> how about we let the namespace return this info in obj.owner, and then we can really consider how to store and retrieve this info correctly per ns type.

Checking this

Reply:
@guymguym Just wanted to know, what namespace we should search for this information. The bucket namespace doesn't have this information. Where else to find this.

Reply:
I meant that the s3 op will not make this decision in this module.
Instead this information should be returned from the object_sdk->namespace_xx level so that every namespace can implement this as desired.
Currently all namespaces can return the bucket owner information, but I can see how this can be enhanced later to allow different owners per object.

Reply:
Makes sense :)